common/src/java/com/zimbra/common/service/ServiceException.java

```diff
@@ -293,6 +293,10 @@ public static ServiceException FAILURE(String message, Throwable cause) {
         return new ServiceException("system failure: "+message, FAILURE, RECEIVERS_FAULT, cause);
     }
 
+    public static ServiceException ERROR_MESSAGE(String str, Throwable cause){
+        return new ServiceException("error: "+str, null, SENDERS_FAULT, cause);
```

Comment:
instead of `str` use some name for var based on the purpose

Reply:
changed the variable name to "message"